### ISSUE TITLE
Enable Recording of Session

### ISSUE THREAD

**Issue by mullender** [Genymobile/scrcpy#125]
It would be useful to be able to record the video of the session to disk. And to be able to match the video to the logcat timestamps.

Perhaps simply running `adb shell screenrecord --bugreport /sdcard/out.mp4` in parallel to `scrcpy` would be good enough.

**Comment by rom1v** [comment]
(closing as duplicate of https://github.com/Genymobile/scrcpy/issues/21).

**Comment by rom1v** [comment]
This is implemented in v1.5:

    scrcpy --record file.mp4

:tada:
